kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/DateTimeConstants.java

```diff
@@ -0,0 +1,45 @@
+package io.delta.kernel.internal.util;/*
```

Comment:
weird package / copyright

Reply:
My intellij formatting settings are messed up for new files.